qa/pull-tester/rpc-tests.py

```diff
@@ -38,112 +39,113 @@
     # Scripts that are run by the travis build process
     # Longest test should go first, to favor running tests in parallel
```

Comment:
Why does the order matter, if it's going to be randomised?

Reply:
This comment is from upstream, and mattered for internal parallelisation when the dominant cost was length of test (where starting the longer tests first is better) rather than computational throughput (where running multiple tests at once that try to use all cores causes contention).